app/controllers/v0/decision_review_evidences_controller.rb

```diff
@@ -1,9 +1,33 @@
 # frozen_string_literal: true
 
+require 'decision_review_v1/utilities/logging_utils'
+
 # Notice of Disagreement evidence submissions
 module V0
   class DecisionReviewEvidencesController < ApplicationController
     include FormAttachmentCreate
+    include DecisionReviewV1::Appeals::LoggingUtils
```

Comment:
including a v1 thing in a v0 controller feels wrong. It works obviously but wonder if we can just not put a version ont the utils class maybe or something

Reply:
Hmm, actually, I think I'm going to leave this as-is. The only other sensible place I can think of to put this module would be the `lib/decision_review` directory which I'm hoping to, at some point, deprecate in favor of the `decision_review_v1` directory.

Reply:
We could also deprecate `decision_review_v1` in favor of `decision_review` as part of our work to upgrade LH APIs across DRs.